src/components/ui/input/index.tsx

```diff
@@ -42,7 +42,7 @@ interface IProps
   showPasswordToggle?: boolean;
   data?: IData;
   inputClassName?: string;
-  onEnterKey?: () => {};
```

Comment:
Shouldn't the right hand side also be `noop` in this line?

Reply:
no since it is a type with void return 
 1. ()=>void -> function which return void
 2. () => {} -> function which return empty object // throw an error in strict type comparision
 
 
![image](https://user-images.githubusercontent.com/104364298/204883744-a0682b09-8548-4958-9473-b14885e43749.png)